.travis.yml

```diff
@@ -33,10 +34,13 @@ before_install:
       brew install yajl;
       brew install augeas --HEAD;
       brew install dbus;
-      brew install qt5;
+      # Building Qt5 currently takes too much time.
```

Comment:
> If the bottled version of Qt 5 is better than the homebrew one we can still change.

The bottled version of Qt 5 is the one installed via Homebrew! Homebrew uses analogies from beer brewing:

| Usual Term             | Homebrew Term |
| ---------------------- | ------------- |
| Package (Description)  | Formula       |
| Prebuilt Package       | Bottle        |
| Repository             | Tap           | 

. Popular Homebrew formulas also provide bottles. So does the Qt 5 formula. However, the Qt 5 formula only provides bottles for Qt without D-Bus support.

> Did the qt-gui work for you?

Yes.

> Even with dbus? (`kdb global-mount dbus`)

Currently `brew install qt5 --with-dbus` does not work on my machine, since I use the latest version of macOS (macOS 10.12.1). I am afraid I can not test this, unless I install Qt from the official [disk image](http://download.qt.io/official_releases/qt/5.7/5.7.0/qt-opensource-mac-x64-clang-5.7.0.dmg).

> When you are logged in you can restart it in the travis GUI. I think we had the discussion once in #160. (I also did not find the button)

Thank you for the information.